src/main/main.ts

```diff
@@ -20,8 +23,10 @@ export function importOnestopToBasecamp(): void {
 
     for(const eventRow of eventRows) {
         if(hasId(eventRow)) {
+            console.log(`Row for ${eventRow.what.value} on ${eventRow.startTime} already exists! Processing it as an existing row...\n`);
             processExistingRow(eventRow);
         } else {
+            console.log(`Row for ${eventRow.what.value} on ${eventRow.startTime} is new! Processing it as a new row...\n`)
```

Comment:
For logging, we need to use `Logger.log()` for Google Apps Scripts

Reply:
Actually, I think console.log still works, but yeah I agree, for consistency, we should just use Logger.log

Reply:
I updated all the instances of console.log to use Logger